### PR TITLE
Add interpolation strategy controls

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: export.py [hunk 1]
```diff
@@ -640,7 +640,7 @@ def create_mesh(ob, scene):
     # elif is_subd_displace_last(ob):
     #    ob.modifiers[len(ob.modifiers)-2].show_render = False
     #    ob.modifiers[len(ob.modifiers)-1].show_render = False
-    mesh = ob.to_mesh(scene, True, 'RENDER', calc_tessface=True,
+    mesh = ob.to_mesh(scene, True, 'RENDER', calc_tessface=False,
```
Comment (bsavery) [inline export.py]: I thought I already set that to false??

File: properties.py [hunk 2]
```diff
@@ -1892,6 +1892,16 @@ class RendermanMeshGeometrySettings(bpy.types.PropertyGroup):
...
+    interp_boundary = IntProperty(
+        name="Subdivision Edge Interpolation Mode", 
+        description="Defines how a subdivided mesh interpolates its boundary edges", 
+        default=1, 
+        min=0, max=2)
+    face_boundary = IntProperty(
+        name="Subdivision UV Interpolation Mode", 
+        description="Defines how a subdivided mesh interpolates its UV coordinates", 
+        default=1, 
+        min=0, max=3)
```
Comment (bsavery) [inline properties.py]: So the default for RfM is 3 here....

File: export.py (hunk 1 above)
Comment (jdent02) [inline export.py]: Guess not?  If it's still considering that a line change.

File: properties.py (hunk 2 above)
Comment (jdent02) [inline properties.py]: Okay.  I can change that.

**Comment by jdent02** [comment]
The docs seemed to indicate that mode 1 was the default.

**Comment by jdent02** [comment]
Changed facevarying default to 3

File: export.py (hunk 1 above)
Comment (bsavery) [inline export.py]: Weird.  Well that's a huge change, basically I think blender was exporting the subdivided faces to the addon and then we were exporting them as a subdiv.  Which is ridiculous!

**Pull request merged**